### ISSUE TITLE
Search functionality broken in built docs.

### ISSUE THREAD

**Issue by mdickinson** [enthought/traits#487]
After building documentation with Sphinx 1.8.5 and the enthought sphinx theme installed, any attempt to enter something in the search box and search returns no results.

Possibly related: sphinx-doc/sphinx#5460

I'm able to manually solve this by editing `search.html` in the built documentation to add the line:
```html
    <script type="text/javascript" src="_static/searchtools.js"></script>
```
just above the line
```html
    <script type="text/javascript" src="_static/language_data.js"></script>
```

**Comment by mdickinson** [comment]
It turns out this is an issue with the Enthought Sphinx Theme: there's a fix in https://github.com/enthought/enthought-sphinx-theme/pull/8

Once that's in, we need to update to the newest EST. The current Sphinx configuration could also do with some cleanup, to make sure that we're not using the local template or static files with the Enthought Sphinx Theme.

**Comment by mdickinson** [comment]
The underlying issue is now fixed in enthought-sphinx-theme 0.6.1 (available on PyPI). We still need to build and upload new working documentation.

**Comment by mdickinson** [comment]
> we need to update to the newest EST

Not necessary: we're currently `pip install`ing directly from GitHub, with no version number specified, so we automatically pick up the latest enthought-sphinx-theme.

**Comment by mdickinson** [comment]
Fixed by #490.
